File: server/ggGURPS.Models/Campaigns/Campaign.cs
using System.Collections.Generic;

public class Campaign
{
    public long Id { get; set; }
    public string Name { get; set; }
    public long GameMasterId { get; set; }
    public GameMaster GameMaster { get; set; }
    public ICollection<Character> Characters { get; set; }
    public Campaign(long id, string name)
    {
        Id = id;
        Name = name;
        Characters = new List<Character>();
    }
}

File: client/src/models/Campaign.ts
interface ICampaign {
    id: number;
    name: string;
    gameMasterId: number;
}

export default class Campaign implements ICampaign {
    public id: number;

    public name: string;

    public gameMasterId: number;

    constructor() {
        this.id = 0;
        this.name = '';
        this.gameMasterId = 0;
    }
}